vowpalwabbit/parse_regressor.cc

```diff
@@ -291,15 +293,17 @@ void save_load_header(
 
           if (!read)
           {
-            msg << all.triples[i] << " ";
-            memcpy(triple, all.triples[i].c_str(), 3);
+            // Use write because all.triples[i] is vector of uint8_ts
```

Comment:
Add a comment explaining null characters can be written to the stringstream with `write()` and what might happen in that case

Reply:
Just updated, can you check if its ok?